frontend/src/pages/Configure/Components.js

```diff
@@ -525,7 +525,7 @@ function RootVolume({basePath, errorsPath}) {
       Volume Type:
       <Select
         disabled={editing}
-        placeholder="Default (gp2)"
+        placeholder="Default (gp3)"
```

Comment:
I might be missing the context here -- but if the user doesn't actually select anything from this dropdown, then the value doesn't end up in the config -- is this correct?

If that's the case then we have a discrepancy -- without setting it explicitly then the true default depends on the version of PC (>3.2 it's gp3 otherwise gp2). So we can update this placeholder based on the PC version, or we can always be explicit and make gp3 be default regardless of the version of PC.

Reply:
(It seems to be set explicitly in the EBS settings)